src/components/input/examples/Inputs.vue

```diff
@@ -172,7 +172,7 @@
         <span id="errorMessage">you have added too many characters, remove some</span>
```

Comment:
Good catch. I fixed the `undefined-error`.  It looks like `cdr-form-error` isn't structured so that the `id` of the span containing the error message has the same value as `aria-errormessage` on the `input` element. Rather it's referencing the div containing the span. Is that ok or do we need the id to be on the element that contains the text node of the error message itself?

Was this test run via `npm test`? I saw a11y tests pop up but I'm having chromedriver issues.

Reply:
No I manually tested this in Axe dev tools - you may need a license for that still, if so we can reach out to Eric

Reply:
Yes it does looks like I need a license for Axe Dev Tools. Last time I used it it was free but rather clunky IIRC

Reply:
OH and this will need to be a unique ID, not "errormessage" but maybe {{input-lable}}-errormessage or something?

Reply:
Yes I fixed that. I got a trial account of axe and it looks like the issue isn't popping up. Will double check after office hours